utils/constants.ts

```diff
@@ -0,0 +1,2 @@
+// production wallet for 0xmacro.eth
+export const TREASURY_ADDRESS = "0x705a47ebc6fce487a3c64a2da64ce2e3b8b2ef55";
```

Comment:
Please triple check and confirm that this is the correct treasury address

Reply:
This is correct 💯